app/src/main/java/com/markkolenbrander/capstonenewsapp/database/converters/DataConverter.kt

```diff
@@ -1,21 +1,22 @@
 package com.markkolenbrander.capstonenewsapp.database.converters
 
 import androidx.room.TypeConverter
+import com.google.gson.Gson
 import com.google.gson.reflect.TypeToken
-import com.markkolenbrander.capstonenewsapp.App
 import com.markkolenbrander.capstonenewsapp.models.Source
 
-
 class DataConverter {
 
+    private val gson = Gson()
```

Comment:
I know we were trying to figure out how to do this with Hilt and inject a `Gson` instance.  From what I've seen online with other examples this seems like the best/simplest approach. Nice work!